docker-compose.yml

```diff
@@ -156,13 +156,14 @@ services:
       - db
       - prometheus
     environment:
-      - DATABASE_SERVICE_NAME=postgres
-      - POSTGRES_SERVICE_HOST=db
-      - POSTGRESQL_DATABASE=compliance_dev
-      - POSTGRESQL_TEST_DATABASE=compliance_test
-      - POSTGRESQL_USER=insights
-      - POSTGRESQL_PASSWORD=insights
-      - SETTINGS__PROMETHEUS_EXPORTER_HOST=prometheus
+#      - DATABASE_SERVICE_NAME=postgres
+#      - POSTGRES_SERVICE_HOST=db
+#      - POSTGRESQL_DATABASE=compliance_dev
+#      - POSTGRESQL_TEST_DATABASE=compliance_test
+#      - POSTGRESQL_USER=insights
+#      - POSTGRESQL_PASSWORD=insights
+#      - SETTINGS__PROMETHEUS_EXPORTER_HOST=prometheus
+      - ACG_CONFIG=/app/test.json
```

Comment:
:clap: this is way better than duplicating env variables. Can we name it based on the devel env, like devel.json? Also, what does ACG mean? Also, let's definitely remove stuff during the PR cleanup instead of just commenting it out.

Reply:
The `ACG_CONFIG` is the variable environment where the Clowder config library expects to read the JSON config file that Clowder injects into a pod it manages. 

Maybe it's `Application ConfiG` ? or something :) - better ask @psav or @kylape about the historical naming reasons :)

Reply:
I also removed the comments and renamed the file to `devel.json`